client/package.json

```diff
@@ -34,7 +34,7 @@
     "start": "vite --host --port 3000",
     "build": "vite build --out-dir build",
     "check-lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --ignore-path ../.gitignore && tsc --noEmit",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
+    "lint": "npm run check-lint --fix",
```

Comment:
At least on windows, the --fix option does nothing so `"npm run check-lint --fix"` == `"npm run check-lint"`

Reply:
Oh... Reverted it back!